File: alumno1/05_funciones/05_funcion_factura.py

cantidad=0
precio=0.0
total=0.0

#ingresar datos
#procesar datos
#mostrar resultados

def ingresar_datos():   
    global cantidad,precio,total

    cantidad= int(input('ingresar cantidad: '))
    precio= float(input('ingresar precio: '))

def procesar_datos():   
    global cantidad,precio,total

    total=cantidad * precio

def mostrar_resultados():   
    global cantidad,precio,total   
    print('Total: ',total)

ingresar_datos()
procesar_datos()
mostrar_resultados()

File: instructor/99_js/05_funciones/05_funcion_factura.js
let cantidad=0
let precio=0.0
let total=0.0

//1) Ingresar Datos
//2) Procesar Datos
//3) Mostrar Resultados

function ingresar_datos() {
    //global cantidad,precio,total

    cantidad = parseInt(prompt('Ingresar Cantidad: '))
    precio = parseFloat(prompt('Ingresar Precio: '))
}

function procesar_datos(){
    //global cantidad,precio,total

    total=cantidad * precio
}

function mostrar_resultados(){
    //global cantidad,precio,total
    console.log('Total=',total)
}

ingresar_datos()
procesar_datos()
mostrar_resultados()